packages/eslint-plugin-pf-codemods/src/rules/helpers/getNodeName.ts

```diff
@@ -0,0 +1,23 @@
+import { JSXOpeningElement, JSXMemberExpression } from "estree-jsx";
+
+/** Gets the name of an opening element or member expression */
+export function getNodeName(node: JSXOpeningElement | JSXMemberExpression) {
```

Comment:
Going off of Adam's comment above, another nit so not a blocker but if this helper is meant solely for these 2 scenarios, might be worth updating the name to reflect that (if we can without it become tooooo verbose).